smart-contracts/osmosis/contracts/cl-vault/tests/test-tube/autocompound.rs

```diff
@@ -54,6 +56,8 @@ fn test_autocompound_with_rewards_swap_non_vault_funds() {
         )
         .unwrap();
 
+    // Get the initial vault token shares LP tokens supply right after the vault deployment
```

Comment:
Could you remove this comment? In general we should try to comment only when absolutely necessary, as
* if they are necessary, then it might be better to refactor the code to be more readable
* often they comment the obvious (i.e. the naming in the following lines contains the same information as this comment)
* often they are misleading or wrong, the code is the real truth, so better let that document itself. I.e. here we the comment says we compare the burnt amount with the initial token supply, but the code shows that for some reason there is a factor 2 (that is the part that should be clarified as it looks sketchy)

Reply:
you right, done!